ros/api/v1/recommendation_ratings.py

```diff
@@ -5,46 +5,90 @@
 from ros.lib.utils import identity, user_data_from_identity
 
 
-class RecommendationRatingsApi(Resource):
+def validate_rating_post_api(func):
+    """Validate POST rating request."""
+    allowed_choices = [c.value for c in RatingChoicesEnum]
 
-    rating_fields = {
-        'inventory_id': fields.String,
-        'rating': fields.Integer
-    }
+    def error_msg(error_code, value):
+        errors = {
+            400: "is invalid value for rating.",
+            422: "is invalid choice of input for rating."
+        }
+        return (
+            f"'{value}' { errors.get(error_code, 'Invalid') }"
+            f"Possible values - { *allowed_choices, }"
```

Comment:
```suggestion
            f" Possible values - { *allowed_choices, }"
```

Reply:
Or, shall we remove L19?
The openapi.json has an enum with a list of choices

Reply:
I would prefer to keep it in error response.